### ISSUE TITLE
[Feature]  Copy to Clipboard option which copies relevant tidal.com link(s) to clipboard.

### ISSUE THREAD

**Issue by blasf69** [exislow/tidal-dl-ng#132]
### Current Situation

I thought of a feature request to help with a problem I have with the app. It might be related to something with my system but download speeds are very slow even while using command line. Yes, 'Download Delay' is turned off. 
The other app, tidal-dl, seems to download 3x-5x faster compared to this app (tidal-dl-ng). 
However, I like this app's GUI and search function. So my idea is to use search to find
the url, then switch over to a tidal-dl command line window and paste the url.
If there's a way to increase speeds comparable to the other app that would be optimal. 
Sorry about the excessive length.

### Suggestion / Feature Request

In the playlist section, I thought it might be useful to add a "Copy to Clipboard" button or a right click "copy url" option, which would copy the playlist tidal link to clipboard.
In the search section, for example, while searching for an album, select/highlight your choice. Use the new button and it will copy the specific url (for example: https://tidal.com/browse/album/113118878 ) to clipboard for later use.
Or maybe adding a right click, context menu / copy option would be easier but since I did not see any right click functionality already implemented (in the search section), a button might be easier to code?

**Comment by exislow** [comment]
You have not provided sufficient information for a comparison regarding the download speed. Anyway, I thought already about a feature, which let you copy the items share link. But this attribute is not present in the used library. I raised an issue in their repo: https://github.com/tamland/python-tidal/issues/266

Let's wait for their thoughts.

As a workaround or even better idea: Why don't you simply use the "Copy Share Link" functionality of the official TIDAL app?

**Comment by blasf69** [comment]
Regarding download speed. I did some comparisons today and speeds were blindingly fast. So I'm not sure what was slowing things down for me. My apologies for bringing this up.

As for not using the official app, I've had issues with tidal working on my phone so I've avoided using it. Mainly just searching Google with site:tidal.com . 

I'd prefer this app's search function over tidal's anyhow. Thanks for replying.
